### PR TITLE
SIP-TBD - Optimism standard deposit/withdrawal interface

### PR DESCRIPTION
**Pull request by ajsantander**
Proposal to update our bridges to conform to the standard interface being proposed by Optimism.

### CODE REVIEW

File: sips/sip-115.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+### Rationale
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.
```
Comment (jacko125) [inline sips/sip-115.md:41]: @ajsantander Should this be removed / uncommented?

File: sips/sip-115.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+sip: 115
```
Comment (justinjmoses) [inline sips/sip-115.md:2]: can you use 116? :pray:

File: sips/sip-115.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+title: Optimism standard deposit/withdrawal interface
+status: WIP
+author: Alejandro Santander (@ajsantander), Yannis TODO (@i-stam)
+discussions-to: https://research.synthetix.io/t/optimism-standard-deposit-withdrawa-interface/322
+
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
```
Comment (justinjmoses) [inline sips/sip-115.md:8]: please add this

File: sips/sip-115.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+requires (*optional): <SIP number(s)>
```
Comment (justinjmoses) [inline sips/sip-115.md:9]: this can be removed

File: sips/sip-115.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+Migrate the existing L1 and L2 bridges to conform to Optimism's standard deposit/withdrawal interface. This standard defines the the function names and signatures for depositing tokens into L2, as well as for withdrawing them back to L1.
+
+## Abstract
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done. If the SIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
+The proposed changes involve renaming a few functions and events in two of our contracts, namely SynthetixBridgeToOptimism and SynthetixBridgeToBase, as well as adding minor functionality to them to be able to deposit and withdraw SNX for another address. The contracts will need to be redeployed, and SynthetixBridgeToOptimism's SNX balance will beed to be migrated to the new L1 bridge contract.
```
Comment (justinjmoses) [inline sips/sip-115.md:20]: [np] please use backticks for contract names

File: sips/sip-115.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+<!--This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.
```
Comment (justinjmoses) [inline sips/sip-115.md:37]: this line is not needed

**Review by justinjmoses — CHANGES_REQUESTED** [review]
Some minor changes 🙏🏼

**Comment by ajsantander** [comment]
Deprecating in favor of https://github.com/Synthetixio/SIPs/pull/405

**Pull request closed without merging**